app/views/admin/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render partial: 'admin' %>
+<h1> gowno </h1>
```

Comment:
If you want to make this project your business card please won't add words like this :P